jina_cli/autocomplete.py

```diff
@@ -203,6 +204,7 @@
             '--version',
             '--loglevel',
```

Comment:
why are these changes related?

Reply:
let me check this come from our best friend jina bot I believe

Reply:
I think with upstream dependencies and more args added in jcloud, jina bot infers it and adds it to our autocomplete

Reply:
here they added logout https://github.com/jina-ai/jcloud/pull/84

Reply:
good catch